PyMPDATA/options.py

```diff
@@ -142,7 +145,11 @@ def __eq__(self, other):
 
     @property
     def n_halo(self) -> int:
-        """halo extent for a given options set"""
+        """Halo extent for a given options set.
+        The halo extent is the number of 'ghost' cells that need to be added
+        to the outside of the domain to ensure that the solution is computed correctly.
+        It is similar to padding in convolutional neural networks.
```

Comment:
worth adding a reference here?

Reply:
Couldn't find a good reference of a paper that describes halo as padding. Maybe you have the link?

Reply:
well, we could drop the reference to convolutional neural networks, and simply state that it is akin to array padding, and refer to https://numpy.org/doc/stable/reference/generated/numpy.pad.html